test/run_test.py

```diff
@@ -137,13 +137,6 @@ def run_test(executable, test_module, test_directory, options):
 
 
 def test_cpp_extensions(executable, test_module, test_directory, options):
-    try:
```

Comment:
i think you should still keep this warning. run_test.py is run by a lot of the devs locally as well